src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -343,45 +341,47 @@ export const VideoTileBase: React.FC<VideoTileBaseProps> = ({
             </TextContainer>
           </CSSTransition>
         </SwitchTransition>
-        <KebabMenuButtonIcon
-          onClick={(event) => openContextMenu(event, 200)}
-          variant="tertiary"
-          size="small"
-          isActive={contextMenuOpts.isActive}
-        >
-          <SvgGlyphMore />
-        </KebabMenuButtonIcon>
-        <ContextMenu contextMenuOpts={contextMenuOpts}>
-          {publisherMode ? (
+
+        <ContextMenu
+          placement={'bottom-end'}
```

Comment:
Unnecessary brackets

Reply:
@kdembler Just an idea, maybe we could enable this rule? https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md 
I also tend to forget about this. Of course, not as part of this PR, wdyt?